packages/client/test/integration/dataunion/calculate.test.ts

```diff
@@ -24,14 +26,17 @@ describe('DataUnion calculate', () => {
         const adminClient = new StreamrClient(clientOptions as any)
 
         const dataUnionName = 'test-' + Date.now()
-        // eslint-disable-next-line no-underscore-dangle
-        const dataUnionPredicted = adminClient._getDataUnionFromName({ dataUnionName, deployerAddress: adminWalletMainnet.address })
+
+        // @ts-expect-error
```

Comment:
A good idea is to add a short description after `@ts-expect-error` to indicate to the future reader why it exists in the first place.

E.g.
```
// @ts-expect-error accessing private method
```